src/Components/ProjectForm.js

```diff
@@ -171,8 +238,11 @@ function ProjectForm() {
                   onChange={handleTagsChange}
                   fullWidth
                 />
-
-            <button data-testid="submit_button" onClick={handleSubmit}>Submit</button>
+            <br /><br />
+            <Button variant="contained" size="large" data-testid="submit_button" onClick={()=>{handleSubmit();handleRoute()}} >
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8488139)
```suggestion
            <Button variant="contained" size="large" data-testid="submit_button" onClick={()=>{handleSubmit();handleRoute();}} >
```